SignalServiceKit/src/Loki/Utilities/XCTest+Utilities.swift

```diff
@@ -0,0 +1,44 @@
+import XCTest
+
+extension XCTestCase {
+
+    /// A helper for asynchronous testing.
+    ///
+    /// Usage example:
+    ///
+    /// ```
+    /// func testSomething() {
+    ///     doAsyncThings()
+    ///     eventually {
+    ///         /* XCTAssert goes here... */
+    ///     }
+    /// }
+    /// ```
+    ///
+    /// The provided closure won't execute until `timeout` seconds have passed. Pass
+    /// in a timeout long enough for your asynchronous process to finish if it's
+    /// expected to take more than the default 0.1 second.
+    ///
+    /// - Parameters:
+    ///   - timeout: number of seconds to wait before executing `closure`.
+    ///   - closure: a closure to execute when `timeout` seconds have passed.
+    func eventually(timeout: TimeInterval = 0.1, closure: @escaping () -> Void) {
+        let expectation = self.expectation(description: "")
+        expectation.fulfillAfter(timeout)
+        self.waitForExpectations(timeout: 60) { _ in
```

Comment:
Wait what's this 60 seconds here?

Reply:
Not sure, i got this file from a gist :(

Reply:
So basically this is just the max amount of time it's going to wait meaning there's sort of a hidden requirement that `timeout` is less than 60 seconds. I added a note :)